.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
_(arbitrary comment location to start a thread)_

> Think this resolves https://github.com/ScopeLift/railgun-shielder/issues/35 , what do you think?

Seems it only asks for my signature the first time in a session, so I'd say yes